src/main/java/com/github/argon/sos/moreoptions/game/booster/MoreOptionsBooster.java

```diff
@@ -0,0 +1,78 @@
+package com.github.argon.sos.moreoptions.game.booster;
+
+import game.boosting.BSourceInfo;
+import game.boosting.BoosterImp;
+import game.faction.FACTIONS;
+import game.faction.Faction;
+import game.faction.npc.NPCBonus;
+import game.faction.npc.ruler.Royalty;
+import init.race.POP_CL;
+import init.race.Race;
+import settlement.army.Div;
+import settlement.stats.Induvidual;
+import world.regions.Region;
+
+public class MoreOptionsBooster extends BoosterImp {
+
+    private double value = 0.0;
+
+    public MoreOptionsBooster(BSourceInfo bSourceInfo, double from, double to, boolean isMul) {
+        super(bSourceInfo, from, to, isMul);
+    }
+
+    public void set(double value) {
+        this.value = value;
+    }
+
+
+    @Override
+    public double vGet(Region reg) {
+        return value / 10000;
```

Comment:
The value 10000 is the overall slider range? because the range is dynamic and can be configured in the MoreOptions.txt.

Reply:
It is, i'll think about how to pass this value

Reply:
fixed it